CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
+---
+function App:connectDebugger()
```

Comment:
You don't need to do this now, just a thought:

It would be really nice if this function was callable as soon as humanly possible. It could still be available for calls from the main menu etc, but at some point you might need to debug from the very beginning of the application. Then this would need to be done either in the App constructor, or at the beginning of App:init. It requires the config to be optional and the check for lua socket to lie earlier though.